lib/collections.js

```diff
@@ -30,7 +36,19 @@ var uniqueWordsAsKeys = {};
 // The keys pulled out into a list
 var uniqueWords = [];
 
-/* your code starts here */
+for (var i = 0; i < normalizedWords.length; i++) {
+  uniqueWordsAsKeys[normalizedWords[i]] = true;
+}
+
+console.log(uniqueWordsAsKeys);
+
+var i = 0;
+for (var word in uniqueWordsAsKeys) {
+  uniqueWords[i] = word;
+  i++;
+}
```

Comment:
Yep, this is a valid answer (though I think `.push` would be more idiomatic in this kind of situation).